### PR TITLE
update support url

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <764524258@qq.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

update overdue url.

```release-note
NONE
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,10 +68,10 @@ That said, if you have questions, reach out to us
 [announcement]: https://cncf.io/news/announcement/2015/07/new-cloud-native-computing-foundation-drive-alignment-among-container
 [Borg]: https://research.google.com/pubs/pub43438.html
 [CNCF]: https://www.cncf.io/about
-[communication]: https://git.k8s.io/community/communication
-[community repository]: https://git.k8s.io/community
+[communication]: https://github.com/kubernetes/kubernetes/blob/master/SUPPORT.md
```
Comment (neolit123) [inline README.md]: i don't think the communication and support pages have the exact same purpose?
is there a tracking issue for this change?

@kubernetes/sig-contributor-experience-pr-reviews

File: README.md [hunk 2]
```diff
@@ -68,10 +68,10 @@ That said, if you have questions, reach out to us
...
+[community repository]: https://kubernetes.io/docs
 [containerized applications]: https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/
-[developer's documentation]: https://git.k8s.io/community/contributors/devel#readme
+[developer's documentation]: https://kubernetes.io/docs/contribute
```
Comment (neolit123) [inline README.md]: i think the same applies for the other 2 link changes.

**Review by neolit123 — COMMENTED** [review]
thanks @daixiang0 
ideally SIG contribex should provide feedback on this change.
you can try pingng them on slack if this PR gets no replies.

/kind documentation
/priority backlog

File: README.md (hunk 1 above)
Comment (daixiang0) [reply README.md]: i just found related pages, maybe not suitable

**Comment by cblecker** [comment]
All of the existing URLs are accurate, and do not need to be changed.
/close

File: README.md (hunk 1 above)
Comment (nikhita) [reply README.md]: Just to add on, the communication link lists all communication platforms that people can use to interact. That page also lists support platforms.

**Comment by daixiang0** [comment]
@cblecker could you visit  https://git.k8s.io/community?

**Comment by cblecker** [comment]
Yes, that works for me. It's a redirector service that redirects to https://github.com/kubernetes/community/

**Pull request closed without merging**